README.md

```diff
@@ -76,48 +90,69 @@ There is nothing :sunglasses: here, no :fireworks: after creating a repository u
 # Technical Requirements
 
 The Generator is a Node.js application. Therefore, template features also depend on Node.js and generator technical requirements:
+
 - Node.js v12.16+
 - npm v6.13.7+
 
 Install both using [official installer](https://nodejs.org/en/download/).
 
-Now install the AsyncAPI Generator globally to use it as CLI:
+Now install the [AsyncAPI Generator](https://github.com/asyncapi/generator) globally to use it as CLI:
+
 ```bash
 npm install -g @asyncapi/generator
 ```
 
 # Template Development Hints
 
 The most straightforward command to use this Template is:
+
 ```bash
 ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.1/test/docs/dummy.yml https://github.com/asyncapi/template-for-generator-templates -o output
 ```
 
 For local development, you need different variations of this command. First of all, you need to know about three important CLI flags:
+
 - `--debug` enables the debug mode in Nunjucks engine what makes filters debugging simpler, 
 - `--watch-template` enables a watcher of changes that you make in the Template. It regenerates your Template whenever it detects a change,
 - `--install` enforces reinstallation of the Template
 
 There are two ways you can work on template development:
+
 - Use global Generator and Template from your local sources:
+
   ```bash
   # assumption is that you run this command from the root of your Template
   ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.1/test/docs/dummy.yml ./ -o output
   ```
+
 - Use Generator from sources and Template also from local sources. This approach enables more debugging options with awesome `console.log` in the Generator sources or even the Parser located in `node_modules` of the Generator:
+  
   ```bash
   # assumption is that you run this command from the root of your Template
   # assumption is that generator sources are cloned on the same level as the Template
   ../generator/cli.js https://raw.githubusercontent.com/asyncapi/generator/v1.0.1/test/docs/dummy.yml ./ -o output
   ```
 
+# Template render engine
+
+The [Generator](https://github.com/asyncapi/generator) supports feature called templating engine. You can write template using tool which you prefer more. The template engine at the moment can be either [React](https://github.com/asyncapi/generator-react-sdk) (recommended) or [Nunjucks](https://mozilla.github.io/nunjucks/) (default). This can be controlled with the `renderer` property in the [template configuration](#renderer).
```

Comment:
What about this?

```suggestion
The [Generator](https://github.com/asyncapi/generator) allows you to write templates using [React](https://github.com/asyncapi/generator-react-sdk) (recommended) or [Nunjucks](https://mozilla.github.io/nunjucks/) (default). This can be controlled with the `renderer` property in the [template configuration](#renderer).
```